src/EventManagement.Domain/Registration.cs

```diff
@@ -183,15 +183,15 @@ private List<OrderLine> _createOrderLines(IEnumerable<Product> products, IEnumer
 						ProductVariantId = v?.ProductVariantId,
 						Price = v?.Price ?? p.Price,
 						VatPercent = v?.VatPercent ?? p.VatPercent,
+						Quantity = p.MandatoryCount, 
```

Comment:
Will this add 0 if is not mandatory?

Reply:
@losviko yes. If MandatoryCount is 0, the quantity will be 0
it needs to be 1 correct?

Reply:
Then it needs 1, yes!

Reply:
..to be..